cmd/openshift-install/create.go

```diff
@@ -169,6 +170,13 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 			if err != nil {
 				return err
 			}
+
+			if os.Getenv("USE_CONFIG_DRIVE") == "True" {
```

Comment:
can we rename CONFIG_DRIVE to IGNITION_FORCE_USE_CONFIG_DRIVE or something similar to make it more specific?

Reply:
Sure~